### PR TITLE
fix(user management): fix disable button state when deselect all checkboxes [updated]

### PR DESCRIPTION
**Pull request by saadanzari**
## Description

Users/Admins are able to deselect all roles assigned to them and confirm during role updates.

Users should not be able to confirm the role update after deselecting all roles assigned to them. At least one role should remain selected at all times.

Users should not be able to confirm the role update after deselecting all roles assigned to them. At least one role should remain selected at all times.

## Why

Please include an explanation of why this change is necessary as well as relevant motivation and context. List any dependencies that are required for this change.

## Issue

https://github.com/eclipse-tractusx/portal-frontend/issues/919

## Checklist

- [x] I have performed a self-review of my own code
- [x] I have successfully tested my changes locally

### CODE REVIEW

**Review by oyo — APPROVED** [review]

**Comment by evegufy** [comment]
Hi @saadanzari thank you for your contribution! I needed to release the release candidate for the 2.1.0 version, could you please solve conflict in the with the changelog?

File: CHANGELOG.md [hunk 1]
```diff
@@ -45,6 +45,9 @@
   - fixed app category view more and collapse button function [#805](https://github.com/eclipse-tractusx/portal-frontend/pull/805)
 - **Connector Management**:
   - fixed blank page issue on clicking on details [#925](https://github.com/eclipse-tractusx/portal-frontend/pull/925)
+- **User Management**:
+  - Fix disable button state when deselect all checkboxes [#923](https://github.com/eclipse-tractusx/portal-frontend/pull/923)
```
Comment (evegufy) [inline CHANGELOG.md:49]: Please add your change to "Unreleased" not under the already released 2.1.0-RC1, thank you:

## Unreleased

- **User Management**:
  - Fix disable button state when deselect all checkboxes [#923](https://github.com/eclipse-tractusx/portal-frontend/pull/923)

## 2.1.0-RC1

...

**Review by evegufy — CHANGES_REQUESTED** [review]
Please see comment

File: CHANGELOG.md (hunk 1 above)
Comment (saadanzari) [reply CHANGELOG.md]: I have updated changelog for bugfix for (unreleased) RC2

File: CHANGELOG.md [hunk 2]
```diff
@@ -46,6 +54,7 @@
 - **Connector Management**:
   - fixed blank page issue on clicking on details [#925](https://github.com/eclipse-tractusx/portal-frontend/pull/925)
 
+
```
Comment (evegufy) [inline CHANGELOG.md:57]: could you please remove the empty line?

**Review by evegufy — CHANGES_REQUESTED** [review]

**Review by evegufy — APPROVED** [review]

**Pull request merged**